src/multi_vector_simulator/A0_initialization.py

```diff
@@ -387,6 +380,13 @@ def process_user_arguments(
     if save_png is None:
         save_png = args.get(SAVE_PNG, DEFAULT_MAIN_KWARGS[SAVE_PNG])
 
+    # if the default input file does not exist, use package default input file
+    if (
+        path_input_folder == DEFAULT_INPUT_PATH
+        and os.path.exists(os.path.join(path_input_folder, JSON_FNAME)) is False
+    ):
+        path_input_folder = os.path.join(PACKAGE_PATH, INPUT_FOLDER)
+
```

Comment:
Will there be an error message/warning if a specified input file is not found the default is used? That would be crucial.

Reply:
This code is executed only if the path_input_folder is the default one (i.e. user did not provide it) and it happens that this path_input_folder does not exist in the system. So in that the path_input_folder is not "specified". I added this to prevent the code to crash if a very lazy user just tries to type `mvs_tool` in the console.

Reply:
But we can add a warning that "default path not found, creating example folder"

Reply:
I would like that.

Reply:
Or "No inputs provided, using the default input path to create the example folder."